python-package/xgboost/sklearn.py

```diff
@@ -3,6 +3,8 @@
 """Scikit-Learn Wrapper interface for XGBoost."""
 from __future__ import absolute_import
 
+from builtins import enumerate
```

Comment:
Entire sklearn.py is enhancement that @arnocandel did, fixing-up and properly testing beyond https://github.com/dmlc/xgboost/pull/2354